ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -948,12 +948,13 @@ TFLiteCore::init (tflite_option_s *option)
   interpreter->setModelPath (option->model_file);
   interpreter->setExtDelegate (option->ext_delegate_path, option->ext_delegate_kv_table);
   num_threads = option->num_threads;
+  int err;
 
   setAccelerator (option->accelerators, option->delegate);
   g_message ("accl = %s", get_accl_hw_str (accelerator));
 
-  if (loadModel ()) {
-    ml_loge ("Failed to load model\n");
+  if ((err = loadModel ())) {
+    ml_loge ("Failed to load model (TensorFlow-lite interpreter->loadModel() has returned %d. Please check if the model, '%s', is accessible and compatible with the given TensorFlow-lite instance. For example, this TensorFlow-lite's version might not support the given model.\n", err, option->model_file);
```

Comment:
how about adding current tf-lite version using predefined values?
(TFLITE_VERSION_MAJOR / MINOR / MICRO)

Reply:
TFLITE_VERSION_* is determined at build time.
However, users CAN replace it at runtime if we allow to use tf-lite shared library.
Thus, using TFLITE_VERSION_* macro is NOT safe. The version should be determined at run-time.

Reply:
I think [TfLiteVersion()](
https://github.com/tensorflow/tensorflow/blob/2afc306a29ce217244784dde30d2171fac46e482/tensorflow/lite/c/c_api.h#L96) can be used but I'm not sure our tflite library supports this.